### ISSUE TITLE
Environment not being read correctly

### ISSUE THREAD

**Issue by davidssmith** [forio/julia-studio#164]
When I run Julia from the command line, it duplicates my shell environment perfectly.  When I run it from Julia Studio, I get a minimal environment that is missing many variables.

For example, I can't run python commands using `PyCall` because I'm using Anaconda.  The path to Anaconda is being set in my shell, but Julia Studio is not reading it.  I put my path in both my `.login` and my `.zshrc`.

Julia Studio is correctly defining `ENV["SHELL"]` as `/bin/zsh`, but it doesn't seem to be doing anything with that information.

**Comment by WestleyArgentum** [comment]
Thanks for the heads up, we'll look into fixing this. Unfortunately I don't have a good workaround right now :/

**Comment by kgoldfeld** [comment]
Is this related to an issue I am having with setting the proper path for packages?  For some reason, the path for my packages defaults to a network drive (H:) and I cannot install packages in that location.  I modified the file prepare-julia-env.bat to include a command "set JULIA_PKGDIR=C:\users\myname\juliapackages."  This works fine when I run Julia as a standalone.  However, when running Julia Studio, that package directory is still pointing to the H:\ drive.

**Comment by WestleyArgentum** [comment]
Quite possibly, though I thought we used `julia.bat` on windows, which should run `prepare-julia-env.bat`. @keesvp might be able to comment more on that.

In the meantime, the workaround there is running `ENV["JULIA_PKGDIR"] = "C:\users\yourname\juliapackages"` inside the console. Lame, I know, but we'll work on this.

**Comment by kgoldfeld** [comment]
Yes - I forgot to mention that I did get it running fine with ENV["JULIA_PKGDIR"] = "C:\users\yourname\juliapackages", but I am really seeking a longer term (less lame) solution.  Any further insight you have or fix would be much appreciated.  Thanks.

**Comment by keesvp** [comment]
The file prepare-julia-env.bat is strictly a Windows and Julia Studio thing. Julia will not look at it. And Julia Studio only accesses it at the first time that Julia is prepared (when you see the message "This may take a while, please be patient...") I will look into what I can see for Windows regarding environment issues.
